crates/hc_bundle/src/cli.rs

```diff
@@ -467,6 +479,24 @@ pub async fn get_dna_name(manifest_path: &Path) -> HcBundleResult<String> {
     Ok(manifest.name())
 }
 
+/// Finds the dna file from the given path, expects to find the DnaManifest if the filename is not given.
```

Comment:
I get what this is saying now. The `get_dna_name` is really doing a "get DNA name from manifest".

Assuming a `dna.yaml` is present is valid when bundling because it is required to create the `.dna` bundle. I don't think it's necessary for the `.dna` file to be in the same directory as the `dna.yaml` for this command to work.

I'd suggest a `walkdir` or similar here to look for a single `.dna` file in the given directory. If 0 or more than 1 `.dna` file is found then that's an error but otherwise it can go ahead with the discovered file. Multiple found files could print the possible paths the user might have wanted to use to stderr or something like that.

`hc dna pack` supports an `output` parameter which can place the bundle elsewhere than the directory containing the `dna.yaml`. So although the scaffolding doesn't use it, we can't assume that these stay in the same directory.

Reply:
The default expectation is that the user will enter the path to the .dna file (which could be anywhere)
so i have removed functions which try to locate the .dna file